realsense/common/win/common_utils.cc

```diff
@@ -49,6 +49,18 @@ scoped_ptr<base::ListValue> CreateErrorResult(ErrorCode error,
   return create_results.Pass();
 }
 
+scoped_ptr<base::ListValue> CreateDOMException(const std::string& message,
+                                               ErrorName name) {
```

Comment:
If the parameters order does not have special concern, just like `(ErrorName name, const std::string& message)` will  facilitate our future work to replace our existing calls of  `CreateErrorResult` with `CreateDOMException`? But that way needs your rework here...